# topology_info.msg
string vehicle_id
uint8 generation_level
bool is_assigner  
string[] my_children
uint8 max_generation